cli.js

```diff
@@ -92,33 +95,33 @@ st.once('testserver', function (server) {
 
 	setState('download');
 	var ping = Math.round(server.bestPing);
-	stats.ping = (cli.flags.json) ? ping : chalk.cyan(ping + chalk.dim(' ms'));
+	stats.ping = cli.flags.json ? ping : chalk.cyan(ping + chalk.dim(' ms'));
 });
 
 st.on('downloadspeedprogress', function (speed) {
 	if (state === 'download' && cli.flags.json !== true) {
-		var download = roundTo(speed, speed > 10 ? 0 : 1);
-		stats.download = chalk.yellow(download + chalk.dim(' Mbps'));
+		var download = roundTo(speed * multiplier, speed > 10 ? 0 : 1);
```

Comment:
When looking one last time into the code I noticed one more thing. The `roundTo` statement is not correct. I think it should be 

```js
speed = speed * multiplier;
var download = roundTo(speed, speed > 10 ? 0 : 1);
```

Because, when you have `20 Mbps`, it's `2.5 MBps`. At the moment, it will be printed as `3 MBps` because the original speed (in bits) is higher then 10.